packages/raystack/components/data-view-beta/data-view.module.css

```
@@ -0,0 +1,320 @@
+.toolbar {
+  padding: var(--rs-space-3) 0;
+  align-self: stretch;
+
+  border-bottom: 0.5px solid var(--rs-color-border-base-primary);
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)